build/source/elmfire_level_set.f90

```diff
@@ -1599,6 +1644,8 @@ SUBROUTINE UX_AND_UY_ELLIPTICAL(L, ACCELERATION_FACTOR, ISTEP, T_ELMFIRE)
 END SUBROUTINE UX_AND_UY_ELLIPTICAL
 ! *****************************************************************************
 
```

Comment:
In the above routine keep the commented out diagnostics but delete the "! DWI ADD 1 -" etc. lines

Reply:
Done it